prometheus-formula/prometheus/files/prometheus-rules.yml

```diff
@@ -100,7 +100,15 @@ groups:
       description: Prometheus {{$labels.instance}} is not ingesting samples.
       summary: Prometheus is not ingesting samples.
     expr: |
-      rate(prometheus_tsdb_head_samples_appended_total{job="prometheus"}[5m]) <= 0
+      (
+        rate(prometheus_tsdb_head_samples_appended_total{%(prometheusSelector)s}[5m]) <= 0
```

Comment:
`%(prometheusSelector)s` part is not defined in this context.

Reply:
Sorry, I missed replacing the upstream preprocessor variable. Fixed by latest commit.